windows/setup/handle_vmci_known_issue.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# On 8.0.1 <= ESXi version <= 8.0.3 with build lower than ESXi 8.0 Update 3c
+# Build 24414501, there is known issue on Windows 11 v24H2 and Windows Server 2025
+# VMs, that guest OS may hang after reverting to 'BaseSnapshot' when VM VBS is enabled.
+#
+- name: "Get VM VBS enable status"
+  include_tasks: ../../common/vm_get_vbs_status.yml
+
+- name: "VM VBS is enabled"
+  when: vm_vbs_enabled
+  block:
+    - name: "Get VMware Tools status"
+      include_tasks: ../../common/vm_get_vmtools_status.yml
+
+    # VMware Tools status will become not running
+    - name: "Refresh VMware Tools running status"
+      community.vmware.vmware_guest_tools_info:
+        hostname: "{{ vsphere_host_name }}"
+        username: "{{ vsphere_host_user }}"
+        password: "{{ vsphere_host_user_password }}"
+        validate_certs: "{{ validate_certs | default(false) }}"
+        datacenter: "{{ vsphere_host_datacenter }}"
+        folder: "{{ vm_folder }}"
+        name: "{{ vm_name }}"
+      register: retry_vmtools_status
+      ignore_errors: true
+      until:
+        - retry_vmtools_status is defined
+        - retry_vmtools_status.vmtools_info is defined
+        - retry_vmtools_status.vmtools_info.vm_tools_running_status is defined
+        - retry_vmtools_status.vmtools_info.vm_tools_running_status == 'guestToolsNotRunning'
+      retries: 60
+      delay: 5
```

Comment:
I think this task can be replaced by `common/vm_wait_vmtools_status.yml`, but the `vmtools_is_running` need to be saved in a temporary variable to use as the when.

Reply:
Yeah, right, thanks.